main.py

```diff
@@ -183,167 +190,122 @@ def compose_message(issues):
+        error_channel = error_handler(channel)
+
+        number_of_issues, conf, author=await modify_params(ctx=ctx, conf=conf, name=name, number_of_issues=number_of_issues, noun=noun, channel=channel, error_channel=error_channel)
+
+        await set_playing('On The Cue')
+        async with channel.typing():
+            issues = [x async for x in get_issues(**conf, error_channel=error_channel)]
+            reply = discord.Embed(color=discord.Color.purple())
+
+            chosen_issues=[]
+            if name=="rand":
+                """
+                @TODO: improve algorithm to save queries
+                1. get first batch of github_max (100), find total number
+                2. if total < requested, return all
+                3. Generate requested random numbers
+                4. get issues till max(generated_list)
+                5. return them
+                """
+                chosen_issues = random.choices(issues, k=number_of_issues)
+                reply.title = f"{number_of_issues} random picks out of {len(issues)}:"
+
+            elif name in ["rq", "fq"]:
+                # the author check is only for review, author=None for feedback
+                if author:
+                    issues = pr_with_pending_review(get_pr_details(issues), author)
+                    # since async islice doesn't exist
+                    async for issue in issues:
+                        chosen_issues.append(issue)
+                        if len(chosen_issues) == number_of_issues:
+                            break
+                else:
+                    chosen_issues=issues[:number_of_issues]
+                kind = "PR(s)" if conf["pull_request"] else "issue(s)"
+                queue_type = "review" if name=="rq" else "feedback"
+                selection = f"for @{author}" if author else f"in {queue_type} queue"
+                reply.title = f"Oldest {number_of_issues} {kind} {selection}:"
+            
+            reply.description = compose_message(beautify_issues(chosen_issues))
+            if len(chosen_issues) < number_of_issues:
+                reply.set_footer(text="There weren't enough...")
+        await channel.send(embed=reply)
+        await set_playing('The Waiting Game')
 
-    reply = discord.Embed(color=discord.Color.purple())
-    reply.description = "Talking to me? Use `!what` to know more."
 
-    if command == 'what':
-        reply.title = "Command list for GitHub Helper"
-        reply.description = '''`!rand <N>`
-Retrieves N random open, non-stale issues
+@bot.command(name='what')
+async def what_cmd(ctx):
+    reply=discord.Embed(color=discord.Color.purple())
+    reply.title = "Command list for GitHub Helper"
+    reply.description = '''`!rand <N> issue/pr`
+Retrieves N random open, non-stale issue(s)/PR(s)
 
-`!review <N>`
-Retrieves N least-recently-updated PR awaiting your review
+`!rq <N> [all]`
+Retrieves N least-recently-updated PR(s) awaiting your review [awaiting general review]
```

Comment:
Slightly confusing description. Can some rewording be done to make it better? Not a blocker.

Reply:
Can you make a suggestion?

Reply:
Retrieves least-recently-updated PR(s) and filters those awaiting a review from you (default) or anyone (in presence of `all`)